Makefile

```
@@ -1,3 +1,7 @@
+ifndef LLAMA_MAKEFILE
+$(error The Makefile build is deprecated. Use the CMake build instead)
```

Might be good to link to https://github.com/ggerganov/llama.cpp/blob/master/docs/build.md here.
```suggestion
$(error The Makefile build is deprecated. Use the CMake build instead. For more details, see https://github.com/ggerganov/llama.cpp/blob/master/docs/build.md)
```